sdk-api-src/content/schannel/ns-schannel-crypto_settings.md

```diff
@@ -90,6 +90,13 @@ If 0, schannel uses system defaults. Set to 0 if the CNG algorithm implies bit l
 
 ## -remarks
 
+The following constant distinguishes between the different RSA padding modes and can be specified in the `strCngAlgId` field, in addition to the <a href="/windows/win32/seccng/cng-algorithm-identifiers">CNG algorithm identifier</a>.
```

Comment:
@Xelu86 - I think this wording sounds good, but do you know what the code would look like if someone passed the CNG algorithm identifier and the padding mode together in the field? It might be useful to show users how to do that. We can move ahead with this change now and circle back to add that example in a separate PR.